rust/arrow/src/record_batch.rs

```diff
@@ -80,6 +80,10 @@ impl RecordBatch {
         Ok(RecordBatch { schema, columns })
     }
 
+    /// Creates a [RecordBatch` from a schema and columns, with additional options,
```

Comment:
```suggestion
    /// Creates a [`RecordBatch`] from a schema and columns, with additional options,
```

Reply:
Thanks, I meant for it to be `RecordBatch`, as the link has little value. I think I removed the wrong character while changing it